### ISSUE TITLE
[Web] Chrome Flags don't work on Flutter 3.3.4

### ISSUE THREAD

**Issue by steniooliv** [flutter/flutter#113628]
Hi guys,

I'm developing an application connecting to an external api.
I was in Flutter version 3.3.2 and to avoid the CORS error I added a flag in the `chrome.dart` file: `"--disable-web-security",`  with this flag I can access the api without errors to run my application in debug mode.
Upgrading to Flutter 3.3.4 this flag stopped working, Chrome doesn't run without restrictions and I can't test my application due to CORS error:

![image](https://user-images.githubusercontent.com/21336806/196513889-6bdc1ebd-e8cf-4078-8297-cddca77298f5.png)

**Comment by jonahwilliams** [comment]
There were no changes to chrome.dart in any of the hotfixes that I'm aware of. If you modify the source of the Flutter SDK tools, you need to re-apply those changes and rebuild the tool after upgrading.

**Comment by steniooliv** [comment]
> There were no changes to chrome.dart in any of the hotfixes that I'm aware of. If you modify the source of the Flutter SDK tools, you need to re-apply those changes and rebuild the tool after upgrading.

I will clear all cache and do the process again, thanks

**Comment by steniooliv** [comment]
Thanks @jonahwilliams  for raising the tool rebuild issue.
I'll describe it in case anyone else has the same problem.

First I edited the  `packages\flutter_tools\lib\src\web\chrome.dart` file , and added at the end of the last argument (`'--disable-translate',`) the `'--disable-web-security'` flag, to allow it to access the external API without being blocked by CORS.

Then I deleted the `flutter\bin\cache\flutter_tools.stamp`  file.

The next time I did a build in chrome, it rebuilt the `flutter_tools.stamp` and successfully applied the arguement.

Thanks everyone, I'll close this
